### DISCUSSION TITLE
JIRA reporter for filing tickets

### DISCUSSION THREAD

**Discussion by huseyin** [ctrf-io/ctrf#19] [Reporters]
I’ve recently enabled Cypress reporter in my project, and it’s been very helpful in pinpointing exact locations of flakiness. But now, I’m struggling to keep track of them. I can definitely create a JIRA ticket myself, but I’m not really in favor of that, especially if you have a few bunches of flaky test suites. I would much rather have it done automatically whenever a flaky failure occurs. Any plans in this regard?

**Comment by Ma11hewThomas** [comment]
Great idea, I've added this to the backlog to be worked on.

**Comment by Ma11hewThomas** [comment]
This has been released - https://github.com/ctrf-io/jira-test-reporter

**Reply by huseyin** [reply]
I will test it out @Ma11hewThomas thanks!
